setup.py

```diff
@@ -14,6 +15,9 @@
     else:
         reqs += [ir]
 
+if os.system('which mpicc') == '0':
```

Comment:
I don't like this.

How about this: https://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies

Reply:
same thing, but makes a long story short: https://stackoverflow.com/questions/10572603/specifying-optional-dependencies-in-pypi-python-setup-py